client/src/routes/admin/AdminPage.tsx

```diff
@@ -14,9 +15,55 @@ import React from 'react';
 import UserContext from '../../context/UserContext';
 import img from '../../assets/poster.png';
 import API from '../../api/apiRoutes';
+import Searchbar from '../../components/Searchbar';
+import { 
+	useNavigate 
+} from 'react-router-dom';
+import {
+	identity, map 
+} from 'cypress/types/lodash';
 
 function AdminPage() : JSX.Element{
 	const { theme } = React.useContext(UserContext);
+	const navigate = useNavigate();
+	
+	const getAllUsers = () => {
+		const result: string[] = [];
+		API('gebruikers').GetAll().then((res)=>{
+			res.data.forEach((e: { naam: any; }) => {
```

Comment:
you can just make state and say that the state = res.data or even better use the useState hook and setState(res.data)